### PR TITLE
3D Object controls

### LINKED ISSUE #6: Controls mechanics (with default unity object) - only visual movement of the joystick and thrust
- [x] #31
- [x] #32
- [x] #33

### LINKED ISSUE #7: Merge 3D modularized model from Blender into Unity with controls mechanics

### PR DESCRIPTION
**Pull request by simonfrayer**
### THE Checklist:
- [x] I have review my code
- [x] I have removed unnecessary comments
- [x] I have merged my **branch** scene into the **main** scene
- [x] I have tested that the app works after merging into the main scene (in Unity is enough)

### CODE REVIEW

File: X Wing VR/Assets/Scripts/CustomJoystickController.cs
Comment (xoxkoo) [inline X Wing VR/Assets/Scripts/CustomJoystickController.cs:1]: are we using this?

**Comment by xoxkoo** [comment]
Folder "Lets make VR Game" lloks like ti can be deleted

**Comment by simonfrayer** [comment]
> Folder "Lets make VR Game" lloks like ti can be deleted

I agree, I will try to move the scripts we are using to ourfolder and delete it

File: X Wing VR/Assets/Scripts/CustomJoystickController.cs
Comment (simonfrayer) [reply X Wing VR/Assets/Scripts/CustomJoystickController.cs:1]: nope, will get rid of it

**Comment by simonfrayer** [comment]
> Folder "Lets make VR Game" lloks like ti can be deleted

Actually, there is quite some stuff we are using and 'might' use in the future, I would be for keeping it now and removing what we ont use later on in the project

**Pull request merged**